test_autogalaxy/analysis/test_plotter_interface.py

```diff
@@ -59,6 +61,22 @@ def test__inversion(
 
     assert path.join(plot_path, "subplot_inversion_0.png") in plot_patch.paths
 
+    with open(path.join(plot_path, "inversion_reconstruction_0.csv"), mode="r") as file:
+        reader = csv.reader(file)
+        header_list = next(reader)  # ['y', 'x', 'reconstruction', 'noise_map']
+
+        reconstruction_dict = {header: [] for header in header_list}
+
+        for row in reader:
+            for key, value in zip(header_list, row):
+                reconstruction_dict[key].append(float(value))

```

Comment:
A similar (although not quite the same) behaviour can be obtained using csv.DictReader

```python
reader = csv.DictReader(file)
list(reader)[0]
> {'y': 1, 'x': 2, ...}